spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -403,4 +409,11 @@ def and_there_should_be_a_change_note(change_note)
     visit step_by_step_page_internal_change_notes_path(@step_by_step_page)
     expect(page).to have_content(change_note)
   end
+
+  def then_I_see_an_unschedule_button
+    within(".publish-or-delete") do
```

Comment:
Any particular reason you're using `within`? I think the test would work without.

Reply:
Similarly, this is how is was previously done https://github.com/alphagov/collections-publisher/blob/3107e52cbcfce5aa46c90df9793591fd24c8b3aa/spec/features/managing_step_by_step_pages_spec.rb#L301-L315
Also I think it in some cases it could be good future proofing.

Reply:
I think it was probably added so that Capybara didn't get confused between clicking on the button, or clicking on the tab of the same name. Not sure if that's still the case!

Again, I may refactor this out later across the whole file - looks good here 👍